x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/SpanFlyout/index.tsx

```diff
@@ -101,11 +109,47 @@ export function SpanFlyout({
           <EuiHorizontalRule />
           <StickySpanProperties span={span} totalDuration={totalDuration} />
           <EuiHorizontalRule />
-          <HttpContext httpContext={httpContext} />
-          <DatabaseContext dbContext={dbContext} />
-          <StackTraceContainer>
-            <Stacktrace stackframes={stackframes} codeLanguage={codeLanguage} />
-          </StackTraceContainer>
+          <EuiTabbedContent
+            tabs={[
+              {
+                id: 'stack-trace',
+                name: 'Strack Trace',
+                content: (
+                  <Fragment>
+                    <HttpContext httpContext={httpContext} />
+                    <DatabaseContext dbContext={dbContext} />
+                    <StackTraceContainer>
+                      <Stacktrace
+                        stackframes={stackframes}
+                        codeLanguage={codeLanguage}
+                      />
+                    </StackTraceContainer>
+                  </Fragment>
+                )
+              },
+              {
+                id: 'tags',
+                name: 'Tags',
+                content: (
+                  <Fragment>
+                    <EuiBasicTable
+                      columns={[
+                        {
+                          field: 'key',
+                          render: (key: string) => <TagName>{key}</TagName>
+                        },
+                        { field: 'value' }
+                      ]}
+                      items={keys(tagContext).map(key => ({
+                        key,
+                        value: get(tagContext, key)
```

Comment:
Do we need `get` here? `tagContext` cannot be `undefined` if we made it in here, right?

Reply:
I think that's because you use `_.keys`. We often lose type info when using lodash. Not saying we have to do it this way, but I _think_ this will preserve type info:
```js
Object.keys(tagContext).map(key => ({
  key,
  value: get(tagContext, key)
}));
```

Reply:
Oh, `tagContext` could be `undefined`. Ok, hmm.. Perhaps 
```js
_.entries(tagContext).map(([key, value])=> ({key, value}))
```

But I guess it doesn't matter that much, so nvm.